src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/HologramWrapper.java

```diff
@@ -0,0 +1,73 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays;
+
+import lombok.CustomLog;
```

Comment:
This import is now unsused, and PMD fails the build

Reply:
Oh I am so sorry! That's what you get for editing files directly on GH and thinking you get away with it. I'll edit it as soon as I get home!